packages/gestalt/src/TableContextProvider.js

```diff
@@ -0,0 +1,14 @@
+// @flow strict
+import { createContext, type Context } from 'react';
+
+type TableContextType = {|
+  stickyColumn?: number,
+|};
+
+const initialState = {
+  stickyColumn: -1,
+};
+
+const TableContext: Context<TableContextType> = createContext<TableContextType>(initialState);
+
```

Comment:
can we export here the Provider rather than the context?

Reply:
can we export useContext as well, useTableContext()

Reply:
+1 – it's a bit unexpected that a file named for the Provider in fact exports the Context

Reply:
I would rename to `TableContext`

and then export both Provider and hook.